client/layout/sidebar/item.jsx

```diff
@@ -68,7 +68,7 @@ export default React.createClass( {
 					<span className="menu-link-text">{ this.props.label }</span>
 					{ isExternalLink ? <span className="noticon noticon-external" /> : null }
 				</a>
-				{ this.renderButton( this.props.buttonLink ) }
+				{ this.renderButton( this.props.buttonLink ) || this.props.children }
```

Comment:
Or, for that matter, can we get rid of `renderButton` altogether if we're not using it? 😆